Mathlib/Data/Real/CauSeq.lean

```diff
@@ -816,13 +816,13 @@ theorem exists_lt (f : CauSeq α abs) : ∃ a : α, const a < f :=
 theorem rat_sup_continuous_lemma {ε : α} {a₁ a₂ b₁ b₂ : α} :
     abs (a₁ - b₁) < ε → abs (a₂ - b₂) < ε → abs (a₁ ⊔ a₂ - b₁ ⊔ b₂) < ε := fun h₁ h₂ =>
   (abs_max_sub_max_le_max _ _ _ _).trans_lt (max_lt h₁ h₂)
-#align cau_seq.rat_sup_continuous_lemma CauSeq.rat_sup_continuous_lemma
+#align rat_sup_continuous_lemma CauSeq.rat_sup_continuous_lemma
```

Comment:
How about this?

Reply:
https://leanprover-community.github.io/mathlib_docs/data/real/cau_seq.html#rat_sup_continuous_lemma